docs/developer-guide/ksqldb-reference/scalar-functions.md

```diff
@@ -971,27 +971,27 @@ JSON_RECORDS("abc") => NULL
 ### `TO_JSON_STRING`
 
 ```sql title="Since: 0.24.0"
-TO_JSON_STRING(val) => String
+TO_JSON_STRING(val) => json_string
```

Comment:
To distinguish from SQL string type.

Reply:
`TO_JSON_STRING` returns an SQL string that contains serialized JSON. I am not sure what is the difference between `json_string` and `String` here.